pages/linux/pidstat.md

```diff
@@ -0,0 +1,23 @@
+# pidstat
+
+> Show system resource usage, including CPU, memory, IO etc.
+
+- Show CPU statistics at two second intervals for 10 times:
```

Comment:
hi @pxgamer @schneiderl  please take a look at this description. I am not sure how to put the placeholder into the description.

Reply:
What about this?
```suggestion
- Show CPU statistics at a given interval a given number of times:
```